workflows/dags/author/author_create/author_create_approved.py

```diff
@@ -130,7 +132,9 @@ def close_author_create_user_ticket(**context: dict) -> None:
     def set_author_create_workflow_status_to_completed(**context: dict) -> None:
         status_name = "completed"
         workflow_management_hook.set_workflow_status(
-            status_name=status_name, workflow_id=context["params"]["workflow_id"]
+            status_name=status_name,
+            workflow_id=context["params"]["workflow_id"],
+            typ=AUTHOR,
```

Comment:
let's keep the naming consistent, this is the `collection_name` right? it's either `literature` or `authors`

Reply:
yes, current its `AUTHORS` and `LITERATURE` as they are constants lmk if I should lower case it